packages/liquidator/index.js

```diff
@@ -196,9 +204,40 @@ async function run({
     const gasEstimator = new GasEstimator(logger);
     await gasEstimator.update();
 
+    const dsProxyManager = new DSProxyManager({
```

Comment:
main change here is that we now need to instantiate the `dsProxyManager` and the `proxyTransaction` wrapper to enable. The latter is fed into the liquidator as a parameter.